### DISCUSSION TITLE
App: clean the log (with a button)

### DISCUSSION THREAD

**Discussion by Gompje** [spatie/ray#96] [Ideas]
ATM I cannot find another way to completely wipe a log (and all pages) other than restarting the app. It would be great to have a button + maybe shortcut to clean the current session completely.

Just File >New would work as wel.

**Comment by freekmurze** [comment]
You can clear the screen using "View > New Blank Screen".

In the preferences, you can also define a hotkey to clear the screen from anywhere.

**Reply by Gompje** [reply]
Yes, but that just creates a new "tab". it does not give you a clean slate. doesn't it?

In my case I was debugging a routine that was being called a LOT and it was flooding the screen. it was running away from me exponentially on a slow dual core with small screen at boot. I did clear the screen during to just keep up but, in the heat of the session, went a screen back -- and I was completely confused what messages I was seeing. 
I was using the → 'new log' thing between runs as well

Think the confusion was caused by the fact that I _thought_ that I had a new blank state, without _any_ history -- while it really wasn't. 

for the record: the closing of the app and the tabs things works is fine for me. I just wanted to share some UX :)

**Reply by Gompje** [reply]
if your wondering: yes it does work on a 2015 macbook 13inch dual core (16ram, m.2 disk) BUT there is a significant delay, its funny in a way even. My poor dual core just cannot keep up! 

(yes I do have another power machine as well)
